io/tests/src/test/java/org/wildfly/extension/io/IOSubsystemTransformerTestCase.java

```diff
@@ -68,11 +67,14 @@ public void testTransformerEAP710() throws Exception {
     }
 
     private KernelServices buildKernelServices(ModelTestControllerVersion controllerVersion, ModelVersion version, String... mavenResourceURLs) throws Exception {
-        return this.buildKernelServices(this.getSubsystemXml(), controllerVersion, version, mavenResourceURLs);
+        String xmlName = String.format("eap-%s-transformer.xml", controllerVersion.getRealVersionName());
```

Comment:
The test fails because there is no `eap-7.0.0-transformer.xml` or `eap-7.1.0-transformer.xml` files. 

I now understand why @ctomc renamed the `io-${subsystem-version}-transformer.xml` to `eap-${eap-version}-transformer.xml` in his PR :)

Reply:
I fixed this using a switch to set up the correct IO version.